ros/demos/chatbot/src/chatbot_node.cpp

```diff
@@ -6,160 +6,33 @@
 
 #include <t_top_hbba_lite/Strategies.h>
 
-
 #include <behavior_srvs/srv/chat_tools_function_call.hpp>
 #include <daemon_ros_client/msg/base_status.hpp>
 #include <std_msgs/msg/u_int8.hpp>
 
+#include <cloud_data/srv/current_local_weather_open_meteo.hpp>  // Weather service definition
```

Comment:
```suggestion
#include <cloud_data/srv/current_local_weather_open_meteo.hpp>
```
The comment does not seem to be useful